src/shared/hooks/useTable.test.ts

```diff
@@ -0,0 +1,66 @@
+import { renderHook, act } from '@testing-library/react';
+
+import { useTable } from './useTable';
+
+jest.mock('redux/modules', () => ({
+  workspaces: {
+    useData: () => ({ ownerId: 'mockOwnerId' }),
+  },
+}));
+
+describe('useTable hook tests', () => {
+  const mockAsyncFn = jest.fn();
+
+  test('useTable should return initial searchValue, order and page', () => {
+    const { result } = renderHook(() => useTable(mockAsyncFn));
+
+    expect(result.current.searchValue).toBe('');
+    expect(result.current.order).toBe('desc');
+    expect(result.current.orderBy).toBe('');
+    expect(result.current.page).toBe(1);
+  });
+
+  test('useTable should reload', () => {
+    const { result } = renderHook(() => useTable(mockAsyncFn));
+    act(() => {
+      result.current.handleReload();
+    });
+
+    expect(mockAsyncFn).toBeCalled();
```

Comment:
it's redundant assertion to check that function was called before `expect(mockAsyncFn).toBeCalledTimes(1);`
please remove here and below.

Reply:
i don't think it's redundant
if some tests fall, it's better to know why - because function wasn't called at all or because it was called more than 1 time.